crates/subspace-networking/src/bin/subspace-bootstrap-node/main.rs

```diff
@@ -48,9 +49,9 @@ enum Command {
         /// Determines whether we allow keeping non-global (private, shared, loopback..) addresses in Kademlia DHT.
         #[arg(long, default_value_t = false)]
         enable_private_ips: bool,
-        /// Defines path for the provider record storage DB (optional).
+        /// Defines path for the directory used for persistent storage of known peers (optional).
         #[arg(long, value_hint = ValueHint::FilePath)]
-        db_path: Option<PathBuf>,
+        storage_dir: Option<PathBuf>,
```

Comment:
I'm open for suggestions on a better name. Should we use `base_path` as in Substrate and that we used in farmer? BTW we don't have to use `base_path` in Substrate.